### DISCUSSION TITLE
pin stories

### DISCUSSION THREAD

**Discussion by WmeLuna** [rodit/SnapMod#189] [Ideas & Feature Suggestions]
add a toggle to pin stories or make them priority show they show up first

**Comment by RevealedSoulEven** [comment]
This is cool too... But I guess these features will make SnapMod very Complicated

**Comment by rodit** [comment]
Added in 1.8.3

**Reply by WmeLuna** [reply]
it stays pinned even after the story is read, would be nice to have that disapear after you saw the story :P
